Initial 1.19 Port

Nilinking opened the pull request:
Mixin changes + Text methods updated

Nilinking committed:
Initial 1.19 Port

mikenrafter reviewed (changes requested):
Looks great! Update that mod version number to indicate it's for 1.19, and it's ready for merging.

mikenrafter committed:
Tweak version to reference 1.19

This may not be perfectly in-line with SemVer, but it's close enough, and gets the point across.
